### PR TITLE
Task s4 create individual product page

### PR DESCRIPTION
**Pull request by o-tahboub**
Created static individual product page.

### CODE REVIEW

File: src/app/individualProduct.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<?php
```
Comment (kyam-mayden) [inline src/app/individualProduct.php:1]: Issue with sidebar on mobile - reduce font size

Comment (kyam-mayden) [reply src/app/individualProduct.php:1]: please

File: src/app/individualProduct.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            <h2>Product Name</h2>
+        </div>
+        <div class="product-content">
+            <div class="row">
+                <div>
+                    <img src="../assets/img/cat2.jpg" class="col-xs-6 col-xs-offset-2 col-sm-4 col-sm-offset-1">
+                </div>
+            </div>
+            <div class="row">
+
```
Comment (kyam-mayden) [inline src/app/individualProduct.php]: whitespace

File: src/app/individualProduct.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                <div class="col-xs-6 col-xs-offset-2 col-sm-4 col-sm-offset-1">
+                    <img src="../assets/img/cat2.jpg" class="img-thumbnail col-xs-6 col-sm-4">
+                    <img src="../assets/img/cat2.jpg" class="img-thumbnail col-xs-6 col-sm-4">
+                    <img src="../assets/img/cat2.jpg" class="img-thumbnail col-xs-6 col-sm-4">
+                </div>
+
```
Comment (kyam-mayden) [inline src/app/individualProduct.php]: whitespace

File: src/app/individualProduct.php [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            </div>
+            <div class="col-xs-6 col-xs-offset-2 col-sm-4 col-sm-offset-1">
+                <div>
+                    <strong class="col-xs-12 col-sm-12">Size</strong>
+                    <input class="col-xs-12 col-sm-12" type="text" name="Size">
+                </div>
+                <div>
+                    <strong class="col-xs-12 col-sm-12">Colour</strong>
+                    <input class="col-xs-12 col-sm-12" type="text" name="Colour">
```
Comment (kyam-mayden) [inline src/app/individualProduct.php]: Should size and color be text or drop down(select and option tags)?

File: src/assets/css/styles.css [hunk 5]
```diff
@@ -1,26 +1,46 @@
-main {
-    margin-top: 100px;
+html, body {
+    height: 100%;
```
Comment (kyam-mayden) [inline src/assets/css/styles.css]: Not sure if this line does anything

**Review by kyam-mayden — CHANGES_REQUESTED** [review]
minor changes to make

File: src/app/individualProduct.php (hunk 1 above)
Comment (o-tahboub) [reply src/app/individualProduct.php:1]: This is done, but had to modify class, please check navbar across 3 pages to confirm this has not broken anything.

File: src/app/individualProduct.php (hunk 2 above)
Comment (o-tahboub) [reply src/app/individualProduct.php]: Removed.

File: src/app/individualProduct.php (hunk 3 above)
Comment (o-tahboub) [reply src/app/individualProduct.php]: Removed.

File: src/app/individualProduct.php (hunk 4 above)
Comment (o-tahboub) [reply src/app/individualProduct.php]: Was not aware there was such a thing, well recommended, changed to select and option tags.

File: src/assets/css/styles.css (hunk 5 above)
Comment (o-tahboub) [reply src/assets/css/styles.css]: Checked, it has no impact on the page, therefore I have removed it.

**Review by kyam-mayden — APPROVED** [review]

**Pull request merged**